plugins/pkg_commands.d/centos

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+. /etc/os-release
+
+# CentOS uses dnf or rpm-ostree(edge images) as the package command
+if [[ -x /usr/bin/rpm-ostree ]]; then
+    RSTRNT_PKG_CMD=${RSTRNT_PKG_CMD:-rpm-ostree}
+    RSTRNT_PKG_CMD=${RSTRNT_PKG_ARGS:-A --idempotent --allow-inactive}
```

Comment:
You assigned the value of `RSTRNT_PKG_ARGS` to `RSTRNT_PKG_CMD`. I guess you wanna use `RSTRNT_PKG_ARGS` on the left side as well.

Reply:
Not sure how that worked in testing, the output must not matter?